docs/content/users/quickstart.md

```diff
@@ -292,6 +292,29 @@ DDEV comes ready to work with any PHP project, and has deeper support for severa
     
     Magento 2 is a huge codebase, and we recommend [using Mutagen for performance](install/performance.md#using-mutagen) on macOS and traditional Windows.
 
+=== "Moodle"
+
+    ## Moodle
+    
+    ```bash
+    ddev config --composer-root=public --create-docroot --docroot=public --webserver-type=apache-fpm
+    mkdir .ddev/php
+    echo "max_input_vars = 5000;" > .ddev/php/custom.ini
+    ddev start
+    ddev composer create moodle/moodle -y
+    ddev exec 'php public/admin/cli/install.php --agree-license  --non-interactive --lang=en --wwwroot=$DDEV_PRIMARY_URL  --dbtype=mariadb --dbhost=db  --dbname=db --dbuser=db --dbpass=db --adminpass=12345 --fullname=DDEVmoodleDemo --shortname=DDEVmoodleDemo'
```

Comment:
```suggestion
    ddev exec 'php public/admin/cli/install.php --non-interactive --agree-license --wwwroot=$DDEV_PRIMARY_URL --dbtype=mariadb --dbhost=db --dbname=db --dbuser=db --dbpass=db --fullname="DDEV Moodle Demo" --shortname=Demo --adminpass=password'
```

I'd prefer to use a minimal set of options here and use `password` for the admin password.

Reply:
I suggested the password. People will understand. This is for introduction to things only.